boa/integrations/jupyter/browser.py

```diff
@@ -2,11 +2,12 @@
 This module implements the BrowserSigner class, which is used to sign transactions
 in IPython/JupyterLab/Google Colab.
 """
-import json
+
 import logging
 import os
 from asyncio import get_event_loop, sleep
 from itertools import chain
+from json import JSONEncoder, dumps
```

Comment:
why?

Reply:
The json keyword is imported from `boa.rpc` to use the faster json when available. But that does not support custom encoders.

Reply:
hmm let's just stick to vanilla `json` module then. i don't think this is particularly performance sensitive